.github/workflows/backport.yml

```diff
@@ -28,7 +28,7 @@ jobs:
         env:
           NUMBER: ${{ github.event.inputs.number }}
           # not using secrets.GITHUB_TOKEN since pull requests from that token do not run workflows
-          GH_TOKEN: ${{ secrets.BOT_TOKEN }}
+          GH_TOKEN: ${{ secrets.OPENTELEMETRYBOT_GITHUB_TOKEN }}
```

Comment:
@trask my understanding is that its safe to use the `OPENTELEMETRYBOT_GITHUB_TOKEN` organization secret for these types of things.

Reply:
yes 👍

I just did the same in https://github.com/open-telemetry/opentelemetry-java-contrib/pull/914